labgrid/remote/client.py

```diff
@@ -1386,8 +1386,7 @@ def main():
         '--crossbar',
         metavar='URL',
         type=str,
-        default=os.environ.get("LG_CROSSBAR", "ws://127.0.0.1:20408/ws"),
-        help="crossbar websocket URL (default: %(default)s)"
+        help="crossbar websocket URL (default: ws://127.0.0.1:20408/ws)"
```

Comment:
I guess we should mention how the default works here:
```suggestion
        help="crossbar websocket URL (default: value from env variable LG_CROSSBAR_REALM, otherwise ws://127.0.0.1:20408/ws)"
```

Reply:
Ok, I've updated the comment